internal/action/render.go

```diff
@@ -51,7 +51,7 @@ func (r Render) Run(ctx context.Context) (*declcfg.DeclarativeConfig, error) {
 		)
 		// TODO(joelanford): Add support for detecting and rendering sqlite files.
 		if stat, serr := os.Stat(ref); serr == nil && stat.IsDir() {
-			cfg, err = declcfg.LoadDir(ref)
+			cfg, err = declcfg.LoadFS(os.DirFS(ref))
```

Comment:
I think it makes sense -- from a testing perspective at least -- to DI this as part of the `Render` struct.

Reply:
Totally agree. I somewhat blindly just replaced `LoadDir()` calls with `LoadFS(os.DirFS())` calls. I'll comb through again to see if there are any other places we can inject from higher levels.

Reply:
Hmm. Upon closer inspection, in the case of `render` it isn't trivial to inject references as filesystems since `render` supports image references, DC directories, and ([probably soon](https://github.com/operator-framework/operator-registry/pull/673)) sqlite DB files.

IMO implementing a containerd fs.FS implementation to read an image reference makes a lot of sense, but I think that's best left to a separate PR.